json-endpoints.php

```diff
@@ -81,13 +82,29 @@
 require_once( $json_endpoints_dir . 'class.wpcom-json-api-get-customcss.php' );
 require_once( $json_endpoints_dir . 'class.wpcom-json-api-update-customcss.php' );
 
+// Logo Settings
+require_once( $json_endpoints_dir . 'class.wpcom-json-api-update-site-logo-endpoint.php' );
+
+// Homepage Settings
+require_once( $json_endpoints_dir . 'class.wpcom-json-api-update-site-homepage-endpoint.php' );
+
+// Publicize
+require_once( $json_endpoints_dir . 'class.wpcom-json-api-publicize-endpoint.php' );
```

Comment:
I think this might have to be moved to .com api endpoint only since this could should only work on the .com side of things. 

Can we remove that part from here.